starttls_policy/tests/util.py

```diff
@@ -0,0 +1,18 @@
+"""Various utils used by tests"""
+
+def parameterized(func):
```

Comment:
this is really not related at all but TIL that parameterize [can be spelled](https://en.wikipedia.org/wiki/Parametrization) "parametrize", and also with an s instead of a z. So there's 4 different spellings that are all valid!

Reply:
Switched to pytest variant of spelling